server/models/Profile.js

```diff
@@ -0,0 +1,19 @@
+const mongoose = require("mongoose");
+
+const ProfileSchema = new mongoose.Schema({
```

Comment:
We already have a Profile model, maybe consider fetching Destin's branch instead? :)